<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HydroFlowForm</class>
 <widget class="QMainWindow" name="HydroFlowForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>290</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>290</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">HydroFlow 1.3</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../hydroflow.qrc">
    <normaloff>:/imagens/hydroflow.gif</normaloff>:/imagens/hydroflow.gif</iconset>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_HidLn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>33</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string> Rede de drenagem </string>
    </property>
    <widget class="QLineEdit" name="lineEdit_HidLn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>381</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_HidLn">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>18</y>
       <width>30</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../hydroflow.qrc">
       <normaloff>:/imagens/search.ico</normaloff>:/imagens/search.ico</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_Exec">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>347</x>
      <y>172</y>
      <width>88</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Executar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Close">
    <property name="geometry">
     <rect>
      <x>347</x>
      <y>230</y>
      <width>88</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Fechar</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_Lim">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>92</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string> Limite da área </string>
    </property>
    <widget class="QLineEdit" name="lineEdit_Lim">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>381</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Lim">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>18</y>
       <width>30</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../hydroflow.qrc">
       <normaloff>:/imagens/search.ico</normaloff>:/imagens/search.ico</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>152</y>
      <width>314</width>
      <height>122</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string> Opções </string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>105</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Unidade do mapa:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Metros</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Quilômetros</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pés</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Outro</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tolerância XY:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_TolXY">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">0.001</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>145</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Inferir:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_Strahler">
     <property name="geometry">
      <rect>
       <x>145</x>
       <y>40</y>
       <width>131</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Ordens por Strahler</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_Shreve">
     <property name="geometry">
      <rect>
       <x>145</x>
       <y>65</y>
       <width>131</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Ordens por Shreve</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_FlowOnly">
     <property name="geometry">
      <rect>
       <x>145</x>
       <y>91</y>
       <width>131</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Somente fluxos</string>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="toolButton_Idioma_pt">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>8</y>
      <width>28</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../hydroflow.qrc">
      <normaloff>:/imagens/Brasil.ico</normaloff>:/imagens/Brasil.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_Idioma_en">
    <property name="geometry">
     <rect>
      <x>423</x>
      <y>8</y>
      <width>28</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../hydroflow.qrc">
      <normaloff>:/imagens/ReinoUnido.ico</normaloff>:/imagens/ReinoUnido.ico</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../hydroflow.qrc"/>
 </resources>
 <connections/>
</ui>
